pkg/engine/engine.go

```diff
@@ -163,7 +163,8 @@ func (e *Engine) alterFuncMap(t *template.Template) template.FuncMap {
 				log.Printf("[INFO] Missing required value: %s", warn)
 				return val, nil
```

Comment:
I think you might actually need to change it here so that it doesn't fail during lint mode.
```suggestion
				return "", nil
```